nep29/nep29.py

```diff
@@ -19,12 +19,13 @@
 from prettytable import PrettyTable, MARKDOWN
 import requests
 from distutils.version import LooseVersion
+from packaging.version import Version
 
 
 def major_minor_version(version):
     """Given a version string, returns the major and minor version a tuple"""
-    split = version.split('.')
-    return int(split[0]), int(split[1])
+    version = Version(version)
+    return version.major, version.minor
```

Comment:
I tried this branch under Python 3.6 with packaging version 19.2 and this line failed because `Version` did not  have the major/minor attributes. As a workaround I used
```suggestion
    return version.release[0], version.release[1]
```
Newer versions of packaging are ok (presumable >= 20.0) but then the dependencies in setup.py should be adjusted accordingly. The workaround works with 20.9.

Reply:
Thanks for the note @orbeckst. Coming back to this is on my to-do list for this weekend/next week. (Currently in the process of transatlantic move. Coincidentally, @orbeckst and I haven't met, but are very much in the same niche [computational chemistry] -- hi there!)

`packaging 19.1` was released on Jul 30, 2019 (a minor version released within the last 24 months), so my opinion is that, assuming the intent is to apply the spirit of NEP 29 to all dependencies here, it should be supported. @hmaarrfk, let me know if your opinion differs.

Exact implementation may differ from @orbeckst's because one of the things I'm aiming for is to ensure is that projects that use epoch versions also work. I'm not sure how that will differ in older versions of packaging (my local branch, based on packaging 20.9, includes epoch support).